libraries/AP_Scripting/docs/docs.lua

```diff
@@ -2389,6 +2389,17 @@ function vehicle:nav_scripting_enable(param1) end
 ---@return boolean
 function vehicle:set_desired_speed(param1) end
 
+-- Set autopilot desired airspeed (Plane)
+---@param airspeed_new number -- new airspeed in m/s
+---@return boolean -- true if successful
+function vehicle:set_desired_airspeed(airspeed_new) end
+
+-- Set the guided mode radius and direction used for the final loiter at the target (Plane)
+---@return boolean -- true if successful
+---@param radius number
+---@param direction_is_ccw boolean
```

Comment:
```suggestion
---@param radius number
---@param direction_is_ccw boolean
---@return boolean -- true if successful
```
Sorry, just spotted this, the return is typically at the bottom.

Reply:
Dammit I broke it.